sentry/src/main/java/io/sentry/hints/Hints.java

```diff
@@ -9,21 +13,51 @@ public final class Hints {
 
   private final @NotNull Map<String, Object> internalStorage = new HashMap<String, Object>();
 
+  public static @NotNull Hints withAttachment(@Nullable Attachment attachment) {
+    @NotNull final Hints hints = new Hints();
+    hints.getAttachmentContainer().add(attachment);
+    return hints;
+  }
+
+  public static @NotNull Hints withAttachments(@Nullable List<Attachment> attachments) {
+    @NotNull final Hints hints = new Hints();
+    hints.getAttachmentContainer().addAll(attachments);
+    return hints;
+  }
```

Comment:
`m`: I'm not sure if we really need to add these convenience methods. The user saves one line of code and we have to add two methods for that. I would rather keep it simple.

Reply:
My intention was to make it as easy as possible to pass an attachment to the `captureX()` methods.